chrome/src/inject/inject.js

```diff
@@ -1,159 +0,0 @@
-"use strict";
```

Comment:
`inject` directory is not required, see `chrome/src/inject.js`

eventually `chrome/src/inject.js` will be `.gitignore`d